### PR TITLE
feat: use Google Tag Manager gtm.js instead of gtag.js

### PR DESCRIPTION
**Pull request by fvsch**
Fixes MKT-69.

This PR updates the analytics snippet code to load both `/gtm.js` and `/gtag/js`.

### CODE REVIEW

File: .vitepress/config.ts [hunk 1]
```diff
@@ -6,21 +6,21 @@ import { defaultGroupLink, sidebarLinks } from '../docs/links';
 
 dotenv.config();
 
-const BASE = '/';
-const BASE_WITH_ORIGIN = `https://developer.stackblitz.com${BASE}`;
+const BASE_PATH = '/';
```
Comment (fvsch) [inline .vitepress/config.ts:9]: 💅 Unrelated cosmetic changes: renamed a constant, and tweaked formatting (running Prettier).

File: .vitepress/config.ts [hunk 2]
```diff
@@ -122,31 +122,28 @@ export default defineConfigWithTheme<ThemeConfig>({
...
+    },
   },
 });
 
 function getAnalyticsTags(env: NodeJS.ProcessEnv): HeadConfig[] {
-  if (!env.VITE_GTM_ID) {
-    return [];
+  const gtmIdPattern = /^GTM-[A-Z\d]+$/;
+  const gtmId = (env.VITE_GTM_ID || '').trim();
+  if (gtmId && gtmIdPattern.test(gtmId)) {
```
Comment (fvsch) [inline .vitepress/config.ts:135]: We check that the `VITE_GTM_ID` environment variable uses the `GTM-` prefix (note that before this change we were using a Google Analytics property id starting with `G-` instead).

**Review by peter-zawistowicz — APPROVED** [review]
✅ 🙏
